app/ota_status.py

```diff
@@ -1,39 +1,29 @@
 #!/usr/bin/env python3
 
+from pathlib import Path
 import tempfile
 import os
 import shutil
 
+import configs as cfg
+from constants import OtaStatusString
 from logging import getLogger, INFO
 
+import constants
+
 logger = getLogger(__name__)
-logger.setLevel(INFO)
+logger.setLevel(cfg.LOG_LEVEL_TABLE.get(__name__, cfg.DEFAULT_LOG_LEVEL))
 
 
 class OtaStatus:
     """
     OTA status class
     """
 
-    # status
-    NORMAL_STATE = "NORMAL"
-    UPDATE_STATE = "UPDATE"
-    PREPARED_STATE = "PREPARED"
-    SWITCHA_STATE = "SWITCHA"
-    SWITCHB_STATE = "SWITCHB"
-    UPDATE_FAIL_STATE = "UPDATE_FAIL"
-    ROLLBACK_STATE = "ROLLBACK"
-    ROLLBACKA_STATE = "ROLLBACKA"
-    ROLLBACKB_STATE = "ROLLBACKB"
-    ROLLBACK_FAIL_STATE = "ROLLBACK_FAIL"
-
-    def __init__(
-        self,
-        ota_status_file="/boot/ota/ota_status",
-        ota_rollback_file="/boot/ota/ota_rollback_count",
-    ):
-        self._status_file = ota_status_file
-        self._rollback_file = ota_rollback_file
+    _status_file = cfg.OTA_STATUS_FILE
+    _rollback_file = cfg.OTA_ROLLBACK_FILE
```

Comment:
It is better to use CAPITAL and underscore if _status_file and _rollback_file are constant.
https://www.python.org/dev/peps/pep-0008/#constants

Reply:
Actually _status_file and _rollback_file are private class attribute,  we use constants from configs.py to configure them, so I think it is OK leave it as it.

Reply:
https://www.python.org/dev/peps/pep-0008/#constants
is a general rule.
When we read a code, and if the value is CAPITAL, we can find that it is read-only.
We can find that it is private since the value name is started with an underscore, but cannot find that it is read-only.
But I think the new PR is OK about this type of fix.

Reply:
This will be fixed along with the PR for new way to distribute configurations.